docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -273,6 +274,17 @@ This demo replaces it with the [Pagination](/components/pagination/) component.
 
 {{"demo": "pages/components/data-grid/rendering/CustomPaginationGrid.js", "bg": "inline"}}
 
+### Toolbar
+
+The grid exposes props to hide specific elements of the UI:
+
+- `hideToolbar`: If `true`, the toolbar component is hidden.
+
+By default, we provide a default toolbar with opinianted functionality. You can replace our default toolbar by providing your own implementation.
```

Comment:
opinianted??

Reply:
I didn't know how to call it. Maybe that is not the best way to describe it, but we will put functionality in it that we think is useful. Maybe just having it as `By default, we provide a default toolbar.` is better.

Reply:
done